corehq/motech/openmrs/repeaters.py

```diff
@@ -225,6 +225,19 @@ def _migration_get_fields(cls):
             "openmrs_config",
         ]
 
+    def _migration_sync_to_sql(self, sql_object):
+        for field_name in self._migration_get_fields():
+            value = getattr(self, field_name)
+            if hasattr(value, 'to_json'):
+                value = value.to_json()
+            if field_name == 'atom_feed_status':
+                value = {
+                    feed_name: feed.to_json
```

Comment:
`feed.to_json` looks like a function. Is it not a function, or is this intentionally constructing a dict of functions? Or should it be `feed.to_json()`?

Reply:
It should be a function call `feed.to_json()` 🤦 Thanks for catching it @millerdev. I wonder how it still works 🤔 
Last refactors are always bad 😅

Reply:
It was Django ORM's internal magic, I guess it calls the attributes while serializing if they are callable.